plugins/web/integration/reference/export-endpoint/step_02.ref

```diff
@@ -1,442 +1,562 @@
 [
   {
+    "_path": null,
+    "_write_ts": null,
     "ts": "2009-11-18T08:00:21.486539",
     "uid": "Pii6cUUq1v4",
-    "id.orig_h": "192.168.1.102",
-    "id.orig_p": 68,
-    "id.resp_h": "192.168.1.1",
-    "id.resp_p": 67,
+    "id": {
+      "orig_h": "192.168.1.102",
+      "orig_p": 68,
+      "resp_h": "192.168.1.1",
+      "resp_p": 67
+    },
     "proto": "udp",
     "service": null,
     "duration": "163.82ms",
     "orig_bytes": 301,
     "resp_bytes": 300,
     "conn_state": "SF",
     "local_orig": null,
+    "local_resp": null,
     "missed_bytes": 0,
     "history": "Dd",
     "orig_pkts": 1,
     "orig_ip_bytes": 329,
     "resp_pkts": 1,
     "resp_ip_bytes": 328,
-    "tunnel_parents": []
+    "tunnel_parents": [],
+    "community_id": null
   },
   {
+    "_path": null,
+    "_write_ts": null,
     "ts": "2009-11-18T08:08:00.237253",
     "uid": "nkCxlvNN8pi",
-    "id.orig_h": "192.168.1.103",
-    "id.orig_p": 137,
-    "id.resp_h": "192.168.1.255",
-    "id.resp_p": 137,
+    "id": {
+      "orig_h": "192.168.1.103",
+      "orig_p": 137,
+      "resp_h": "192.168.1.255",
+      "resp_p": 137
+    },
     "proto": "udp",
     "service": "dns",
     "duration": "3.78s",
     "orig_bytes": 350,
     "resp_bytes": 0,
     "conn_state": "S0",
     "local_orig": null,
+    "local_resp": null,
     "missed_bytes": 0,
     "history": "D",
     "orig_pkts": 7,
     "orig_ip_bytes": 546,
     "resp_pkts": 0,
     "resp_ip_bytes": 0,
-    "tunnel_parents": []
+    "tunnel_parents": [],
+    "community_id": null
   },
   {
+    "_path": null,
+    "_write_ts": null,
     "ts": "2009-11-18T08:08:13.816224",
     "uid": "9VdICMMnxQ7",
-    "id.orig_h": "192.168.1.102",
-    "id.orig_p": 137,
-    "id.resp_h": "192.168.1.255",
-    "id.resp_p": 137,
+    "id": {
+      "orig_h": "192.168.1.102",
+      "orig_p": 137,
+      "resp_h": "192.168.1.255",
+      "resp_p": 137
+    },
     "proto": "udp",
     "service": "dns",
     "duration": "3.75s",
     "orig_bytes": 350,
     "resp_bytes": 0,
     "conn_state": "S0",
     "local_orig": null,
+    "local_resp": null,
     "missed_bytes": 0,
     "history": "D",
     "orig_pkts": 7,
     "orig_ip_bytes": 546,
     "resp_pkts": 0,
     "resp_ip_bytes": 0,
-    "tunnel_parents": []
+    "tunnel_parents": [],
+    "community_id": null
```

Comment:
I'm pondering whether I like that we now have all custom fields present, like `community_id` and `local_resp`. The fact that we have the `--omit-*` functions makes me think it's okay, though.